### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by cs-sameer-pathan**
### Descriptions:
- PMDB: [click here](https://pmdb.fortinet.com/ProjectManagement/editprojects.php?id=13350)

#### UTCs:
- [x] Connector installation verified. 
- [x] Connector logo verified. 
- [x] Actions and Playbooks list verified.
- [x] Playbooks tags verified. 
- [x] All Playbooks are in info mode verified. 
- [x] All Playbooks are in inactive mode verified.
- [x] All actions are tested and verified with required parameters.
- [x] All actions are tested verified with and without optional parameters

### CODE REVIEW

File: fortinet-fortivoice/info.json [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+      "parameters": [
+        {
+          "title": "Upload File",
+          "name": "upload_file",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "file",
+          "tooltip": "Upload license file",
+          "description": ""
```
Comment (cs-harsh-sharma) [inline fortinet-fortivoice/info.json]: Description missing

File: fortinet-fortivoice/info.json [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+      "parameters": [
+        {
+          "title": "Uploaded File ID",
+          "name": "file_id",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "text",
+          "tooltip": "Specify uploaded license file ID to apply to vm. You can get this ID from 'Upload License File' action. ",
+          "description": ""
```
Comment (cs-harsh-sharma) [inline fortinet-fortivoice/info.json]: Description missing

**Review by cs-harsh-sharma — APPROVED** [review]

File: fortinet-fortivoice/info.json [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+        "value": true,
+        "tooltip": "Specifies whether the SSL certificate for the server is to be verified or not.By default, this option is set as True.",
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not.By default, this option is set as True."
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "get_all_devices",
+      "title": "Get All Devices",
```
Comment (cs-prerna-joshi) [inline fortinet-fortivoice/info.json]:
```suggestion
      "title": "Get Devices List",
```

File: fortinet-fortivoice/info.json [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+        {
+          "title": "Get Unassigned Devices",
+          "name": "get_non_assigned_devices",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "checkbox",
+          "tooltip": "Set it to true to retrieve unassigned devices only. By default it is set to false.",
+          "description": "Set it to true to retrieve unassigned devices only. By default it is set to false.",
+          "value": "false"
```
Comment (cs-prerna-joshi) [inline fortinet-fortivoice/info.json]:
```suggestion
          "value": false
```

File: fortinet-fortivoice/info.json [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+        {
+          "title": "Start Index",
+          "name": "start_index",
+          "required": false,
+          "editable": true,
+          "visible": true,
+          "type": "integer",
+          "tooltip": "Specify the start index of devices you want to retrieve from Fortinet FortiVoice.",
+          "description": "Specify the start index of devices you want to retrieve from Fortinet FortiVoice. By default it is set as 0.",
+          "value": "0"
```
Comment (cs-prerna-joshi) [inline fortinet-fortivoice/info.json]:
```suggestion
          "value": 0
```

File: fortinet-fortivoice/info.json [hunk 6]
```diff
@@ -0,0 +1,189 @@
...
+        },
+        {
+          "title": "Size",
+          "name": "size",
+          "required": true,
+          "editable": true,
+          "visible": true,
+          "type": "integer",
+          "placeholder": "i.e., 50",
+          "value": "50",
```
Comment (cs-prerna-joshi) [inline fortinet-fortivoice/info.json]:
```suggestion
          "value": 50,
```

File: fortinet-fortivoice/operations.py [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+        self._server_url = config.get('host', '').strip('/')
+        if not self._server_url.startswith('https://') and not self._server_url.startswith('http://'):
+            self._server_url = 'https://' + self._server_url
+        self._username = config.get('username')
+        self._password = config.get('password')
+        self._verify_ssl = config.get('verify_ssl') if config.get('verify_ssl') else False
+        self.cookie = requests.cookies.cookiejar_from_dict(config.get('cookie'))
+
+    def generate_cookies(self, config):
+        try:
```
Comment (cs-prerna-joshi) [inline fortinet-fortivoice/operations.py]: remove the try except block to avoid the unnecessary stack track.

**Review by cs-prerna-joshi — APPROVED** [review]

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**